### PR TITLE
Finish protected route

### PR DESCRIPTION
**Pull request by DragisaDraskovic**
Finish protected routes

### CODE REVIEW

File: src/utils/ProtectedRoute.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import React from 'react'
+
+import { Navigate } from 'react-router-dom'
+
+import LoginService from '../services/LoginService'
+
+
+const ProtectedRoute = ({ children }: {children: JSX.Element} ) => {
+
+  !LoginService.isLoggedIn()
```
Comment (brankastevic) [inline src/utils/ProtectedRoute.tsx]: What is the purpose of `!LoginService.isLoggedIn()` on line number 11?

File: src/utils/ProtectedRoute.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  if(!LoginService.isLoggedIn()) {
```
Comment (brankastevic) [inline src/utils/ProtectedRoute.tsx]: You can write this in a shorter way, please use the ternary operator in this case.

File: src/App.tsx [hunk 3]
```diff
@@ -1,23 +1,22 @@
...
-
   return (
-    <Router >
+    <Router>
       <Routes>
-        <Route path='/' element={<Login />} />
-        <Route path='/home' element={<MainPage /> } />
+        <Route path='/login' element={<Login />} />
+        <Route path='*' element={<Login/>} />
+        <Route path='/home' element={<ProtectedRoute><MainPage /></ProtectedRoute>} />
```
Comment (vladimir-agbaba-valcon) [inline src/App.tsx]: Home nije private, njoj moze da se pristupi svakako, samo ce u zavisnosti od toga da li je korisnik ulogovan neke opcije biti sakrivene

File: src/utils/ProtectedRoute.tsx (hunk 1 above)
Comment (DragisaDraskovic) [reply src/utils/ProtectedRoute.tsx]: Purpose of this line was testing. I was forgotten to remove it. I will remove it.

File: src/utils/ProtectedRoute.tsx (hunk 2 above)
Comment (DragisaDraskovic) [reply src/utils/ProtectedRoute.tsx]: Ok, I will fix this.

File: src/App.tsx (hunk 3 above)
Comment (DragisaDraskovic) [reply src/App.tsx]: Ok. Razumem. Ispravicu ovo.

**Review by brankastevic — APPROVED** [review]

File: src/App.tsx [hunk 4]
```diff
@@ -1,23 +1,21 @@
+
```
Comment (aleksandarkrnjaic) [inline src/App.tsx]: Why the empty line?

File: src/App.tsx [hunk 5]
```diff
@@ -1,23 +1,21 @@
...
 import React from 'react'
```
Comment (aleksandarkrnjaic) [inline src/App.tsx]: This is , probably, not needed. Try it.

File: src/App.tsx [hunk 6]
```diff
@@ -1,23 +1,21 @@
...
-
   return (
-    <Router >
+    <Router>
       <Routes>
-        <Route path='/' element={<Login />} />
-        <Route path='/home' element={<MainPage /> } />
+        <Route path='/login' element={<Login />} />
+        <Route path='*' element={<Login/>} />
+        <Route path='/home' element={<MainPage />} />
```
Comment (vladimir-agbaba-valcon) [inline src/App.tsx:13]: Are you going to take care of private routes  at this branch or you will do it later?

**Review by aleksandarkrnjaic — APPROVED** [review]

**Review by vladimir-agbaba-valcon — APPROVED** [review]

File: src/App.tsx (hunk 6 above)
Comment (DragisaDraskovic) [reply src/App.tsx:13]: I will do it leter.

File: src/App.tsx (hunk 5 above)
Comment (DragisaDraskovic) [reply src/App.tsx]: Ok, I copied your file.

File: src/App.tsx (hunk 4 above)
Comment (DragisaDraskovic) [reply src/App.tsx]: I didn't saw that.

**Pull request merged**